### ISSUE TITLE
Submit app to app store failed

### ISSUE THREAD

**Issue by HideOnBushTuT** [microsoft/onnxruntime#8237]
**Describe the bug**
i use onnxruntime-mobile-c through cocoapods in iOS App . when i submit app to app store,  it failed.
> ERROR ITMS-90171: "Invalid Bundle Structure - The binary file '****.app/Frameworks/onnxruntime.framework/onnxruntime' is not permitted. Your app can’t contain standalone executables or libraries, other than a valid CFBundleExecutable of supported bundles. Refer to the Bundle Programming Guide at https://developer.apple.com/go/?id=bundle-structure for information on the iOS app bundle structure.

- Mac OS 11.4
- Xcode 12.5
- Swift version 5.4

![image](https://user-images.githubusercontent.com/16773862/123909076-1337ef80-d9ab-11eb-93dc-8372797e79be.png)

**Comment by gwang-msft** [comment]
Thanks for reporting the issue. We will look into this. 
We will need some more information to help the troubleshooting,
- Is there any other error messages when you try to submit to the App store, other than the one you posted above?
- What is your App's deployment target?

**Comment by HideOnBushTuT** [comment]
*  Invalid MinimumOSVersion. Apps that only support 64-bit devices must specify a deployment target of 8.0 or later. MinimumOSVersion in '[My App].app/Frameworks/.framework' is ''."

I solve this problem by importing the corresponding corresponding plist files into the library

* Op with name (Pad_82) and type (Pad) kernel not found in CPUExecutionProvider. No matching hash for 17900876414644757728 
this error occurs when i run the app

**Comment by gwang-msft** [comment]
For this error 
`Op with name (Pad_82) and type (Pad) kernel not found in CPUExecutionProvider. No matching hash for 17900876414644757728`
Can you please share the `required_operators.config` or `required_operators_and_types.config` which was generated when you convert to ONNX model to ort format?

**Comment by HideOnBushTuT** [comment]
Should I open another issue for this error ?

**Comment by gwang-msft** [comment]
Yes, please open another issue for the (Pad) kernel not found error.

**Comment by HideOnBushTuT** [comment]
> Yes, please open another issue for the (Pad) kernel not found error.

see this issue #8266

**Comment by gwang-msft** [comment]
@HideOnBushTuT, we recently merged a new change #8357 make it possible to generate the static framework for onnxruntime, and hopefully this would unblock you, please give it a try.

**Comment by HideOnBushTuT** [comment]
thanks i'll try it.

**Comment by gwang-msft** [comment]
@HideOnBushTuT please try the newly published onnxruntime-mobile-c 1.8.2 on CocoaPods, which should resolve this issue.
If some of the operators in your model is not included in our [pre-built package](https://onnxruntime.ai/docs/reference/mobile/prebuilt-package/1.8%20ORTMobilePackageOperatorTypeSupport), you would need to perform a [custom build](https://onnxruntime.ai/docs/how-to/mobile/custom-build.html)
